AsyncSwift/Views/StampView.swift

```diff
@@ -8,10 +8,78 @@
 import SwiftUI
 
 struct StampView: View {
+    var tempisScanned: Bool = true
+    @State var isStampClicked: Bool = false
```

Comment:
```suggestion
    var tempisScanned = true
    @State var isStampClicked = false
```

Bool은 Swift를 믿고 !  생략하셔도 되지 않을까 싶어요 ~ !